subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -739,21 +760,9 @@ static void http_fota_handler(const struct fota_download_evt *evt)
 
 	case FOTA_DOWNLOAD_EVT_ERROR:
 		LOG_ERR("FOTA_DOWNLOAD_EVT_ERROR");
-		(void)update_job_execution(client_internal,
-					   job_id_handling,
-					   sizeof(job_id_handling),
-					   AWS_JOBS_FAILED,
-					   "");
 
-		aws_fota_evt.id = AWS_FOTA_EVT_ERROR;
-
-		callback(&aws_fota_evt);
-		reset_library();
-
-		/* If the FOTA download fails it might be due to the image being deleted.
-		 * Try to get the next job if any exist.
-		 */
-		(void)aws_jobs_get_job_execution(client_internal, "$next", get_topic);
+		/* Ignore return value, since we can't deal with a failed update here anyways. */
```

Comment:
Why can't we deal with a failed update here? IMO not checking the return value of a function deserves a descriptive comment if its not super obvious.